tests/unit/core/binarycodec/fixtures/data/codec-fixtures.json

```diff
@@ -4435,20 +4435,229 @@
       }
     }
   ],
-  "transactions": [{
-    "binary": "1200002200000000240000003E6140000002540BE40068400000000000000A7321034AADB09CFF4A4804073701EC53C3510CDC95917C2BB0150FB742D0C66E6CEE9E74473045022022EB32AECEF7C644C891C19F87966DF9C62B1F34BABA6BE774325E4BB8E2DD62022100A51437898C28C2B297112DF8131F2BB39EA5FE613487DDD611525F17962646398114550FC62003E785DC231A1058A05E56E3F09CF4E68314D4CC8AB5B21D86A82C3E9E8D0ECF2404B77FECBA",
-    "json": {
-      "Account": "r3kmLJN5D28dHuH8vZNUZpMC43pEHpaocV",
-      "Destination": "rLQBHVhFnaC5gLEkgr6HgBJJ3bgeZHg9cj",
-      "TransactionType": "Payment",
-      "TxnSignature": "3045022022EB32AECEF7C644C891C19F87966DF9C62B1F34BABA6BE774325E4BB8E2DD62022100A51437898C28C2B297112DF8131F2BB39EA5FE613487DDD611525F1796264639",
-      "SigningPubKey": "034AADB09CFF4A4804073701EC53C3510CDC95917C2BB0150FB742D0C66E6CEE9E",
-      "Amount": "10000000000",
-      "Fee": "10",
-      "Flags": 0,
-      "Sequence": 62
+  "transactions": [
+    {
```

Comment:
Why did this first one change?

Reply:
Oh, for the first one I only changed `"transactions": [{` to `"transactions": [` and moved `{` to a new line. Just to make the json indentation better. The actual Payment codec fixture didn't change.